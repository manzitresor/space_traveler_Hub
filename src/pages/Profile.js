import React from 'react';
import ReservedRockets from './reservedRockets';

function Profile() {
  return (
    <>
      <ReservedRockets />

    </>
  );
}

export default Profile;
