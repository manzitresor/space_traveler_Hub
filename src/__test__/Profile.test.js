import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import Profile from '../pages/Profile';
import store from '../redux/store';

describe('Profile page testing', () => {
  it('should render Profile page', () => {
    const profile = render(
      <Provider store={store}>
        <Profile />
      </Provider>,
    );
    expect(profile).toMatchSnapshot();
  });
});
