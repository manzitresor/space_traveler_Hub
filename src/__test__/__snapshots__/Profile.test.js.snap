// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile page testing should render Profile page 1`] = `
<div>
  <div
    class="profile"
    style="display: flex; justify-content: center; gap: 20px; margin: 20px auto;"
  >
    <div
      class="missions"
    >
      <h2
        class="missions-h1"
      >
        My Missions
      </h2>
      <ul
        class="missions-ul"
        style="list-style-type: none; padding: 0px; margin: 0px;"
      />
    </div>
    <div
      class="rockets"
    >
      <h2
        class="rockets-h1"
      >
        My Rockets
      </h2>
    </div>
  </div>
</div>
`;
